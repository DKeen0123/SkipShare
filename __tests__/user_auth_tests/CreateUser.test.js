import React from 'react';
import { shallow } from 'enzyme';
import CreateUser from '../../user/CreateUser';

const props = { type: 'Register' }
const createUser = shallow(<CreateUser {...props} />);

it('renders without crashing', () => {
  expect(createUser).toMatchSnapshot();
});