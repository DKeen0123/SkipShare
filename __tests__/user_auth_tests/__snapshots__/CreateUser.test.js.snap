// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Component>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Register
  </Text>
  <UserForm
    onSubmit={[Function]}
    type="Register"
  />
</Component>
`;
